benchmarks/markdown/gatsby-node.js

```diff
@@ -1,32 +1,32 @@
 const NUM_PAGES = parseInt(process.env.NUM_PAGES || 5000, 10)
-const template = require(`./page-template`)
+const template = require('./page-template')
```

Comment:
looks like prettier isn't handling this correctly. It's different from the other packages inside the monorepo.